assets/models/foundrylocal/mistralai-mistral-7b-instruct-v0.2-trtrtx-gpu/spec.yaml

```diff
@@ -0,0 +1,27 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: mistralai-mistral-7b-instruct-v0.2-trtrtx-gpu
+version: 1
+path: ./
+tags:
+  foundryLocal: "test"
+  license: "Apache-2.0"
+  licenseDescription: "This model is provided under the License Terms available at <https://www.apache.org/licenses/LICENSE-2.0.html>."
+  author: Microsoft
+  inputModalities: "text"
+  outputModalities: "text"
+  task: chat-completion
+  maxOutputTokens: 2048
+  alias: mistralai-mistral-7b-instruct-v0.2
+  directoryPath: mistralai-Mistral-7B-Instruct-v0.2
+  promptTemplate: "{\"system\": \"<s>\", \"user\": \"[INST]\\n{Content}\\n[/INST]\", \"assistant\": \"{Content}</s>\", \"prompt\": \"[INST]\\n{Content}\\n[/INST]\"}"
+type: custom_model
+variantInfo:
+  parents:
+  - assetId: azureml://registries/azureml/models/mistralai-mistral-7b-instruct-v0.2/versions/1
```

Comment:
Change to 'azureml://registries/azureml/models/mistralai-Mistral-7B-Instruct-v0-2/versions/6'